docs/getting-started/setup.mdx

```diff
@@ -0,0 +1,174 @@
+
+    @WidgetbookApp.material()
+    late int notUsed;
+    ```
+
+    3- Execute the `build_runner` by running the following command:
+    ```bash
+    flutter pub run build_runner build
+    ```
+    or 
+    ```bash
+    flutter pub run build_runner watch
+    ```
+
+<Info>You may rename `ui_catalogs.dart` to your preferred name that fits your project.</Info>
+
+The command generates the `ui_catalogs.widgetbook.dart` file.
+
+```dart
+your_flutter_project/
+│─ ...
+├─ lib/
+│ ├─ main.dart
+│ ├─ ui_catalogs.dart
+│ ├─ ui_catalogs.widgetbook.dart
+│ ├─ src/
+├─ pubspec.yaml
+└─ ...
+```
+
+    4- The generated app.widgetbook.dart file should look like this:
+
+    ```dart title="app.widgetbook.dart"
+        // Generated Widgetbook file: app.widgetbook.dart
+        import 'package:flutter/material.dart';
+        import 'package:widgetbook/widgetbook.dart';
+
+        void main() {
+        runApp(HotReload());
+        }
+
+        class HotReload extends StatelessWidget {
+        const HotReload({Key? key}) : super(key: key);
+
+        @override
+        Widget build(BuildContext context) {
+            return Widgetbook.material(
+            addons: [],
+            directories: [],
+            );
+        }
+    }
+    ```
+
+With these steps, you have successfully set up Widgetbook using the generator approach with annotations.
+
+  </TabItem>
+  <TabItem value="manual">
+    1- Create a new file `app.widgetbook.dart` in your `lib` folder.
+
+<Info>You may rename `app.widgetbook.dart` to your preferred name that fits your project.</Info>
+
+```dart
+your_flutter_project/
+│─ ...
+├─ lib/
+│ ├─ main.dart
+│ ├─ app.widgetbook.dart
+│ ├─ src/
+├─ pubspec.yaml
+└─ ...
+```
+
+    2- Add the following code snippet to your `app.widgetbook.dart` file:
+
+    ```dart title="app.widgetbook.dart"
+    // Widgetbook file: app.widgetbook.dart
+    import 'package:flutter/material.dart';
+    import 'package:widgetbook/widgetbook.dart';
+
+    void main() {
+      runApp(const HotReload());
+    }
+
+    class HotReload extends StatelessWidget {
+      const HotReload({Key? key}) : super(key: key);
+
+      @override
+      Widget build(BuildContext context) {
+        return Widgetbook.material(
+          addons: [],
+          directories: [],
+        );
+      }
+    }
+    ```
+
+  </TabItem>
+</Tabs>
+
+## Custom Theme
+
+`Widgetbook` allows developers to define whatever `Theme` they have defined for their app. To accompany every theme and the ones defined by Flutter, the following constructors exist:
+
+| Theme        | Constructor             | Defaults to                      |
+| ------------ | ----------------------- | -------------------------------- |
+| Custom Theme | `Widgetbook<YourTheme>` | ➖                               |
+| Material     | `Widgetbook.material`   | `Widgetbook<ThemeData>`          |
+| Cupertino    | `Widgetbook.cupertino`  | `Widgetbook<CupertinoThemeData>` |
```

Comment:
This might be the wrong location to talk about themes. Let's discuss this in a quick meeting.

Reply:
Moved it to a new page until we write more about htis.

Reply:
Moved it to a new page until we write more about htis.